addons/paint-skew/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Skewing in costume editor",
+  "description": "Hold Ctrl while scaling an object on its sides to skew it. Hold Alt while skewing an object to skew it from its center.",
```

Comment:
Maybe we could clarify that you need to hold BOTH Ctrl and Alt to skew from the center. Or is it clear enough, what do you think?
cc @DNin01 @BroJac5246

Reply:
There are a few ways we could go with this:
- This piece of information could be excluded because you can already hold Alt to resize from the center. If the user knows that, they could theoretically apply that knowledge to the skewing feature.
- Yeah, why not? Doesn't hurt to add a few words to further clarify.
- It does say "while skewing an object", which was explained in the previous sentence. I would find that clear enough within a few seconds.

Honestly, anything works. I can't really decide.

Reply:
Yeah tbh I read the description wrong twice :P 
The first time I tried out this addon I tried skewing from a corner lol

Reply:
Probably more worth it to consider how "skewing" could be translated into other languages. I have no idea how I'll translate it into Spanish tbh.